### PR TITLE
0.2/779 registration seeds break connection between centre and family initialcentre

### PR DESCRIPTION
**Pull request by charlesstrange2**
The seeds were knackered and created impossible views of data.

### CODE REVIEW

File: database/migrations/2017_12_04_113343_create_registrations_table.php [hunk 1]
```diff
@@ -15,7 +15,6 @@ public function up()
     {
         Schema::create('registrations', function (Blueprint $table) {
             $table->increments('id');
-            $table->string('cc_reference')->nullable(); // The Children's Centre system id
```
Comment (katjam) [inline database/migrations/2017_12_04_113343_create_registrations_table.php:18]: Still ok to hack the migrations?

File: database/seeds/RegistrationSeeder.php [hunk 2]
```diff
@@ -12,33 +12,65 @@ class RegistrationSeeder extends Seeder
...
         $family->save();
+
+        // create 3 regs for a *new* centre (with no users)
+        $this->createRegistrationForCentre(3);
+    }
+
+
+    /**
+     * This is horrible and there's a better way to mke seeds, imagine.
+     * @param $quantity
```
Comment (katjam) [inline database/seeds/RegistrationSeeder.php:41]: lol yeah - this is odd.

Comment (katjam) [reply database/seeds/RegistrationSeeder.php:41]: But if it works :ship:

File: resources/views/service/printables/families.blade.php [hunk 3]
```diff
@@ -30,7 +30,7 @@
         @foreach ($chunk as $registration)
         <tr>
           <td>{{ $registration->family->carers->first()->name }}</td>
-          <td>ARC00123</td>
+          <td>{{ $registration->family->rvid }}</td>
```
Comment (katjam) [inline resources/views/service/printables/families.blade.php:33]: Is it called something else now or still rvid?

**Review by katjam — APPROVED** [review]
Yep - my comments are just quips. Nothing to worry about. :)

:st_kitts_nevis:

File: resources/views/service/printables/families.blade.php (hunk 3 above)
Comment (charlesstrange2) [reply resources/views/service/printables/families.blade.php:33]: $family->rvid is now a magical dynamically constructed attribute!

File: database/migrations/2017_12_04_113343_create_registrations_table.php (hunk 1 above)
Comment (charlesstrange2) [reply database/migrations/2017_12_04_113343_create_registrations_table.php:18]: I guess... :-)

**Pull request merged**